### PR TITLE
Lint example html  

### PR DESCRIPTION
**Pull request by gero3**
**Description**

This PR adds the html examples to the linter. This also adds the cleaned up html files.

### CODE REVIEW

File: examples/webxr_ar_dragging.html [hunk 1]
```diff
@@ -55,7 +54,7 @@
 				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 0.1, 10 );
 				camera.position.set( 0, 0, 3 );
 
-				controls = new OrbitControls( camera, container );
+				new OrbitControls( camera, container );
```
Comment (Mugen87) [inline examples/webxr_ar_dragging.html:57]: Even if the linter complains here, I think it's more readable to write this line like so:
```js
const controls = new OrbitControls( camera, container );
```

**Comment by gero3** [comment]
I've update the 3 examples by adding reasonable min and max distances

**Comment by mrdoob** [comment]
Ops, looks like you added the builds by mistake.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**